_layouts/default.html

```diff
@@ -56,6 +56,9 @@
                     <li><a href="{{ root_path }}#quick-start">Quick Start</a></li>
                     <li><a href="{{ root_path }}#download">Download</a></li>
                     <li><a href="//formats.kaitai.io/">Format Gallery</a></li>
+                    {% assign news_prefix = '/news/' %}
+                    {% assign page_url_start = page.url | slice: 0, news_prefix.size %}
+                    <li{% if page_url_start == news_prefix %} class="active"{% endif %}><a href="/news/">News</a></li>
```

Comment:
I've seen how it looks in action and I prefer this order:

```diff
-Kaitai Struct	What is it?	Quick Start	Download	Format Gallery	News
+Kaitai Struct	What is it?	Quick Start	Download	News		Format Gallery
```

This way, format gallery still has a "prominent" place as the last link in the left group. It also makes more sense to me to finish the links related to the core of the project ("Download" and "News" are quite related) and then we can link to the format gallery.

Reply:
Agreed, here is the new layout

![image](https://user-images.githubusercontent.com/9056906/227392705-660402d1-3e6c-4770-b6c0-e904df87f7e5.png)